sw/vendor/BUILD

```diff
@@ -2,4 +2,18 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+package(default_visibility = ["//visibility:public"])
+
 exports_files(glob(["veri-titan/gen/*.s"]))
+
+cc_library(
+    name = "cryptoc_sha256",
+    srcs = [
```

Comment:
Nit: Personally, I like to put single-sources on one line. :)

Reply:
done.